pkg/controller/util.go

```diff
@@ -1109,8 +1145,19 @@ func createImportEnvVar(client kubernetes.Interface, pvc *v1.PersistentVolumeCla
 	podEnvVar := &importPodEnvVar{}
 	podEnvVar.source = getSource(pvc)
 	podEnvVar.contentType = getContentType(pvc)
+	podEnvVar.volumeMode = getVolumeMode(pvc)
```

Comment:
If we remove the envVar variable, this would have to be regular:
```go
volumeMode := getVolumeMode(pvc)
```
And use that variable below.